code/datums/elements/organ_explode_on_removal.dm

```diff
@@ -0,0 +1,23 @@
+/**
+ * ## DANGEROUS ORGAN REMOVAL ELEMENT
+ *
+ * Makes the organ explode when removed fromm the mob through any means
+ */
+/datum/element/dangerous_organ_removal
```

Comment:
we already have ``/datum/element/dangerous_surgical_removal``, should be unified with that